launchable/commands/subset.py

```diff
@@ -187,6 +187,15 @@ def subset(
         )
         sys.exit(1)
 
+    if is_observation and is_output_exclusion_rules:
+        click.echo(
+            click.style(
+                "WARNING: Both options --observation and --output-exclusion-rules are set. "
+                "No output will be generated in this case.",
+                fg="yellow"),
+            err=True,
+        )
```

Comment:
Don't we need to send it as telemetry like this warning case?

Reply:
I added WARNING event 👍